src/app/services/postman-sms/postman-sms.service.ts

```diff
@@ -156,6 +223,137 @@ class PostmanSmsService {
       )
     })
   }
+
+  public sendBouncedSubmissionSms(
+    recipient: string,
+    recipientEmail: string,
+    adminId: string,
+    adminEmail: string,
+    formId: string,
+    formTitle: string,
+  ): ResultAsync<true, SmsSendError | InvalidNumberError> {
+    const logMeta = {
+      action: 'sendBouncedSubmissionSms',
+      formId,
+    }
+
+    logger.info({
+      message: `Sending bounced submission notification for ${recipientEmail}`,
+      meta: logMeta,
+    })
+
+    if (!isPhoneNumber(recipient)) {
+      logger.warn({
+        message: `${recipient} is not a valid phone number`,
+        meta: logMeta,
+      })
+      return errAsync(new InvalidNumberError())
+    }
+
+    const message = renderBouncedSubmissionSms(formTitle)
+
+    const smsData: BouncedSubmissionSmsData = {
+      form: formId,
+      collaboratorEmail: recipientEmail,
+      recipientNumber: recipient,
+      formAdmin: {
+        email: adminEmail,
+        userId: adminId,
+      },
+    }
+
+    return this._sendInternalSms(
+      smsData,
+      recipient,
+      message,
+      SmsType.BouncedSubmission,
+    )
+  }
+
+  public sendFormDeactivatedSms(
```

Comment:
`sendBouncedSubmissionSms`, `sendFormDeactivatedSms`, and `sendAdminContactOtp` is mostly duplicated as well. Can create a common method and inject the differences as parameters

Reply:
It was designed to be a drop-in replacement. 

I do see benefits of us dedup-ing it. I think we can address this after BTN.

Reply:
https://linear.app/ogp/issue/FRM-1742/btn-migration-or-de-dup-internal-flow